### ISSUE TITLE
Failed to set `text-field` property for layer

### ISSUE THREAD

**Issue by Tapestes** [mapbox/mapbox-maps-ios#1595]
## Environment
- Xcode version: 14.0.1
- iOS version: 15.7
- Devices affected: iPad Air 2
- Maps SDK Version: 10.8

## Observed behavior and steps to reproduce
When the following code is used on a SymbolLayer that is a text field rather than icon, we get "Failed to set `text-field` property for `c-airspace-labels` layer. Error: [1]: Expected number but found null instead."
```
try style.updateLayer(withId: "c-airspace-labels", type: SymbolLayer.self) { (layer: inout SymbolLayer) in
     layer.visibility = .constant(sender.isOn ? .visible : .none)
}
```
Code works fine for SymbolLayers that have icons.  Same code has worked fine on the same map previously.   Not sure which version of the SDK caused the issue. Appears similar to [Issue 268](https://github.com/mapbox/mapbox-maps-ios/issues/268)

## Expected behavior

Visibility should be set accordingly.

**Comment by maios** [comment]
Hi @Tapestes, thank you for reporting this issue. We have addressed this bug in https://github.com/mapbox/mapbox-maps-ios/pull/1565 and released [v10.9.0-beta.1](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.9.0-beta.1) recently including this fix.

**Comment by Tapestes** [comment]
Thanks @maios. I can confirm that fixes the issue for me.  Appreciate the help.
